src/components/search/SearchKeywordBox.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import Button from '~/components/common/Button';
+import Input from '~/components/common/Input';
+
+/* Style */
+const SearchKeywordBoxLayout = styled.div`
+  display: flex;
+  flex-direction: row;
+
+  input {
+    width: 20rem;
+
+    margin-right: 1rem;
+  }
+`;
+
+interface SearchKeywordBoxProps {
+  search: (keyword: string) => void;
+  initialKeyword: string;
+}
+
+const ButtonMemo = React.memo(Button);
+
+const SearchKeywordBox: React.FC<SearchKeywordBoxProps> = ({ search, initialKeyword }) => {
+  const [keyword, setKeyword] = useState<string>(initialKeyword);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setKeyword(e.target.value);
+  };
+
+  const handleInputKeyEvent = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.keyCode !== 13) return;
```

Comment:
```suggestion

    const ENTER = 'Enter'
    if (e.key !== ENTER) return;
```
어떠나유 ㅎㅎ

Reply:
@lallaheeee 저 저번에 이렇게 써본거같은데 이럴 경우에 모바일 키보드에서 전송(?) 버튼 누르면 Enter만으로는 작동 안하는걸로 알아요!

Reply:
@lallaheeee 물론 지금은 웹이긴 하지만요...

Reply:
@mango906  오 새로운 사실 알아갑니다 ㅎㅎ 

그렇다면 어떠나유 
```suggestion

    const ENTER = 'Enter'
    if (e.keyCode !== 13 || e.key !== ENTER ) return;
```

Reply:
@lallaheeee   e.key !== ENTER를 추가한데 특별한 이유가 있으신가요??

Reply:
13이라는 숫자가 `enter`라는 걸 알기 힘드니까 
**상수**로 값의 의미를 명확히 기술하여 가독성을 높이면 좋겠다는 생각이 들었어요 ㅎㅎ

```
const ENTER = 13
if( e.keyCode !== ENTER ) 
```
이ㅇㅓ도 좋을것같아여 ㅎㅎ

Reply:
@lallaheeee 좋아요!

Reply:
찾아보니 경빈님이 말씀하신 모바일 키패드 문제도 그렇구 브라우저나 모바일 별로 key code에 대한 이슈가 있는 것 같았어요.
몇몇 사람들은 key code 처리에 대한 polyfill 함수를 만들어서 사용하는 것 같아서 그런 함수를 만드는것도 좋은 것 같아요!

[keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)
[stack overflow related to keyCode](https://stackoverflow.com/questions/35394937/keyboardevent-keycode-deprecated-what-does-this-mean-in-practice)

Reply:
오 이런 디테일 매우 좋아요 ㅋㅋㅋㅋ 
`Enter`가 있다는 것은 처음알았네요 :+1: